### ISSUE TITLE
Portuguese language code handling is totally funky

### ISSUE THREAD

**Issue by jamalex** [learningequality/ka-lite#3931]
The central server turns `pt-BR` into `pt` when building language packs (for some part of the process, at least), but the distributed server doesn't know/handle this well. For now, this has had a bandaid applied here: https://github.com/learningequality/ka-lite/commit/1b7a561afbfe822034acf2973ceb48b42444c489

Initial symptom was that dubbed video mappings were not found (video tree endpoint returned `null` on video download page), and when that was patched, video availability was not stamped in sidebar topic tree, and when that was patched, videos were marked as "not available" once you finally got to the video page. The patches in https://github.com/learningequality/ka-lite/commit/1b7a561afbfe822034acf2973ceb48b42444c489 are a very temporary hack; somebody needs to do a deeper dive and resolve this more systematically.

**Comment by jamalex** [comment]
Temporary fix in https://github.com/learningequality/ka-lite/pull/3930 (no urgency on this now, @aronasorman, as the hacks are working for the time being)

**Comment by aronasorman** [comment]
Thanks @jamalex! I'll try to get to the root of this issue when we do the i18n bug bash.

**Comment by radinamatic** [comment]
I don't know if it's related, but I'm doing a fresh clone/install (today's develop) and I got a lot of error lines like this, after the `generaterealdata` command:  

```
DEBUG:kalite:conflicting entry of dubbed_youtube_id poXtU-URNeY in pt-PT dubbed
video map
DEBUG:kalite:conflicting entry of dubbed_youtube_id MuvZ_Rp_3uc in pt-PT dubbed
video map
DEBUG:kalite:conflicting entry of dubbed_youtube_id 7Uj68wfOgZM in pt-PT dubbed
video map
DEBUG:kalite:conflicting entry of dubbed_youtube_id gUbPpLzvklY in pt-PT dubbed
video map
DEBUG:kalite:conflicting entry of dubbed_youtube_id EGJdPsus5YM in pt-PT dubbed
video map
DEBUG:kalite:conflicting entry of dubbed_youtube_id Y39u1Kh26PQ in pt-PT dubbed
video map
DEBUG:kalite:conflicting entry of dubbed_youtube_id sEnf9by7wQY in pt-PT dubbed
video map
DEBUG:kalite:conflicting entry of dubbed_youtube_id bVlL7D85wO0 in pt-PT dubbed
```

**Comment by aronasorman** [comment]
Fixed in #3930. Let's open another issue when we get another bug related to this.
